testsuite/testcases/src/performance_test.rs

```diff
@@ -15,7 +15,7 @@ impl Test for PerformanceBenchmark {
 
 impl NetworkTest for PerformanceBenchmark {
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
-        let duration = Duration::from_secs(60);
+        let duration = Duration::from_secs(180);
```

Comment:
Curious - why did we increase this?

Reply:
This used to be 5 min, but I reduced it to 1 min to speed up the test. Bumped it back up to 3 min so that we might get lower variance in the test results